doc/conf.py

```diff
@@ -32,7 +32,13 @@
 # ones.
 
 sys.path.insert(0, os.path.join(os.path.abspath('.'), 'extensions'))
-extensions = ['breathe', 'sphinx.ext.graphviz', 'kerneldoc']
+extensions = ['breathe', 'sphinx.ext.graphviz', 'sphinx.ext.extlinks', 'kerneldoc']
+
+# extlinks provides a macro template
+
+extlinks = {'acrn-commit': ('https://github.com/projectacrn/acrn-hypervisor/commit/%s', ''),
```

Comment:
Very cool mechanism these `acrn-commit` and `acrn-issue`.. I like them!! :-)

We should also add them to the doc guidelines.